test/features/resource_freshness.feature

```diff
@@ -27,6 +27,32 @@ Feature: Resource freshness
             | TestOrgAdmin  |
             | TestOrgEditor |
 
+    Scenario Outline: An editor, admin or sysadmin user, when I go to the dataset new page and select 'biennially' update frequency, then the text 'Next update due' should be visible
```

Comment:
Frequently could become a second parameter to the scenario outline, instead of needing separate tests.